x-pack/plugins/graph/public/app.js

```diff
@@ -538,7 +539,10 @@ app.controller('graphuiPlugin', function ($scope, $route, $interval, $http, kbnU
   $scope.saveUrlTemplate = function () {
     const found = $scope.newUrlTemplate.url.search(drillDownRegex) > -1;
     if (!found) {
-      notify.warning('Invalid URL - the url must contain a {{gquery}} string');
+      toastNotifications.addWarning({
+        title: 'Invalid URL',
+        text: 'The url must contain a {{gquery}} string',
```

Comment:
In the description:

url -> URL
add ending period after string

The URL must contain a {{gquery}} string.